src/filter/elastic_scale/elastic_scale.cpp

```diff
@@ -122,20 +118,25 @@ class ElasticScale : public frei0r::filter
     }
 
 private:
-    
+
+    // input params    
+    f0r_param_double m_scaleCenter;
+    f0r_param_double m_linearScaleArea;
+    f0r_param_double m_linearScaleFactor;
+    f0r_param_double m_nonLinearScaleFactor;
```

Comment:
Note that the double and bool param types have recently been changed in all plugins.
This does work for the double type, but using f0r_param_bool would be a mistake. Therefore f0r_param_double is misleading..

Reply:
Changed the data types from f0r_param_double to double